rfc-119-merge-mainstream-browse-topics-and-taxons.md

```diff
@@ -0,0 +1,169 @@
+# RFC 119: Merge mainstream browse, specialist topic into to taxon content items
+
+## Summary
+
+Mainstream browse, specialist topic and taxon content items are functionally equivalent, in that they represent a category that groups content items. We would like to maintain future consistency between our taxonomy, mainstream browse and specialist topic titles. We propose merging the attributes of mainstream browse and topic content items into taxons and deprecating use of mainstream browse and specialist topics in the Publishing API and Content Store.
+
+## Problem
+
+We currently maintain three seperate ways to categorise content items - mainstream browse pages, specialist topics and taxons. There is currently ongoing work to make sure mainstream browse and specialist topics have equivalents in the taxonomy i.e. mainstream browse and specialist topics become a subset of taxons. This is being done by Content Designers who are liaising with departments to come to agreement.
```

Comment:
I think it would be helpful to link to the schemas/examples for these formats and/or explain more about them. Many people may not be aware of what they are, and the fact that they're so similar makes them hard to understand at a glance - another argument for getting rid of them, of course!

Reply:
Yup, how about this?

---

## Background

There are currently three different and independent ways to categorise content items - [mainstream browse pages](https://docs.publishing.service.gov.uk/document-types/mainstream_browse_page.html), [specialist topics](https://docs.publishing.service.gov.uk/document-types/topic.html) and [taxons](https://docs.publishing.service.gov.uk/document-types/taxon.html). A content item can be associated with one or more mainstream browse pages, specialist topics or taxons. Each of which represents a grouping of content items that are related in subject matter. For example content items representing a "Income Tax" guidance page and a "Inheritance Tax" guidance page are assigned to the "Tax" mainstream browse page. 

Mainstream browse pages, specialist topics and taxons themselves are represented as content items. Each type can contain links to other items of the same type and usually do so building a hierarchical tree structure. For example, the "Child benefit" and "Universal Credit" specialist topics are linked and represented as children of the "Benefits" specialist topic. For each type there is one "root" item that represents the start of the hierarchy. There is an exception for taxons, where there are two hierarchies with two distinct roots and sets of items - [topic taxonomy](https://docs.publishing.service.gov.uk/manual/taxonomy.html) and [world taxonomy](https://docs.publishing.service.gov.uk/manual/world-taxonomy.html). For purposes of this RFC ignore the world taxonomy. 

See the [appendix](#appendix) and developer docs for more information and examples.

## Problem

We have to maintain three separate types of categorisation for content items - mainstream browse pages, specialist topics and taxons. This means the categories themselves and the content assigned to them may be different between each type. For example, the "Education and learning" mainstream browse page is the logical equivalent of the "Education, training and skills" taxon and should be the same, however they differ in name and in the set of content items they represent. There are also categories that only exist in some types and not others, for example the "Setting up a business" mainstream browse page has no equivalent taxon. 

There is currently ongoing work to ensure mainstream browse and specialist topics have equivalents in the taxonomy, i.e. mainstream browse and specialist topics become a subset of taxons. This is being done by Content Designers who are liaising with departments to come to agreement.

Once this work is complete we want to maintain alignment and prevent divergence from taxons. We would have to put constraints in collections publisher and content tagger (or any future apps) to ensure that mainstream browse and specialist topics continue to be subsets of taxons. This would involve adding complex logic to the publishing system to deal with addition, changing or removal of these content items.

Reply:
Really helpful, thanks!

Minor: we could inline the three examples in the appendix here, since this is where we introduce them.

I think it would also be helpful to clarify what these three things are for, as the docs we're linking to don't explain, and the examples are just examples. Why do we have these three navigation systems?

Related to that, I'm confused about whether this would lead to any user-facing changes on GOV.UK. Are we proposing to continue rendering existing mainstream browse and specialist topic pages as we are now, but using a different content item? This is relevant for the problem/background section because I'm not sure if we're trying to work towards only having one navigation system, or just combining the attributes of the existing ones.

Reply:
This work itself should not introduce any user facing changes - mainstream browse and topic pages will continue to exist. The content design work to ensure mainstream browse and specialist topics have equivalents in the taxonomy may cause user facing changes. But this RFC is contingent on that work already being done.

Cool - I'll add in my suggested changes with the examples + a note that this shouldn't introduce user facing changes. @WorkSutton do mind you addressing @benthorners comments about adding in more information about what these three navigation systems used for?

Reply:
@theseanything @benthorner Sure I can add some background on the 3 navigation systems based on my current understanding

Reply:
### Mainstream browse
- Currently the `primary` browse navigation of GOV.UK
- 2 level hierarchy containing ~ 150 browseable topoics on GOV.UK homepage
- sub-topics users see as curated lists of Mainstream guides
- users cannot subscribe to email updates

### Specialist topics
- Currently the `secondary` browse navigation on GOV.UK
- 2 level hierarchy containing ~ 410 browseable from department and organisation pages
- sub-topics users see as curated lists of Specialist guids
- users can subscribe to email updates 

### Taxonomy topics
- Very few users utilize this to navigate the site
- multi-level hierarchy of variable depth (1-5 levels) comprising ~ 1300 topics
- reachable from breadcrumbs and related links
- content of all types is tagged to every level of the hierarchy
- uncurated
- users can subscribe to email updates

Reply:
Thanks for the summary @WorkSutton. Definitely some new insights in there for me, and I reckon others would find it helpful for this to be incorporated in the RFC.

I'm not sure what `primary` and `secondary` mean, though. I can see that Mainstream Browse Categories are the ones listed on the homepage (so perhaps `primary` in the sense that they're the main way for users to browse GOV.UK content). Where do we surface the Specialist Topic pages that makes them `secondary`?

Reply:
`primary` and `secondary` is here is a shorthand for the fact that in terms of navigational prominence, traffic, and effort - mainstream topics are currently prioritised.

Specialist topics are surfaced on organisation pages, related links and breadcrumbs.